packages/headless/src/features/search/search-actions.ts

```diff
@@ -289,7 +291,16 @@ export const buildSearchRequest = (
       facets: getFacets(state),
     }),
     ...(state.fields && {
-      fieldsToInclude: state.fields.fieldsToInclude,
+      fieldsToInclude: [
+        ...state.fields.fieldsToInclude,
+        ...(state.folding?.enabled
+          ? [
+              state.folding.fields.collection,
+              state.folding.fields.parent,
+              state.folding.fields.child,
+            ]
```

Comment:
Should the folding fields be added here? Or should dispatching `registerFolding` add the folding fields to `fieldsToInclude`? The latter feels more correct, but it might not matter if `fieldsToInclude` is only read inside `buildSearchRequest`.